src/Api/PubnubApi/EndPoint/Objects/GetChannelMembersOperation.cs

```diff
@@ -1,240 +1,265 @@
+		{
+			PNResult<PNChannelMembersResult> returnValue = new PNResult<PNChannelMembersResult>();
+
+			PNPageObject internalPage;
+			if (page == null) { internalPage = new PNPageObject(); } else { internalPage = page; }
+			RequestState<PNChannelMembersResult> requestState = new RequestState<PNChannelMembersResult>();
+			requestState.ResponseType = PNOperationType.PNGetChannelMembersOperation;
+			requestState.Reconnect = false;
+			requestState.UsePostMethod = false;
+			requestState.EndPointOperation = this;
+
+			var requestParameter = CreateRequestParameter();
+			Tuple<string, PNStatus> JsonAndStatusTuple;
+			var transportRequest = PubnubInstance.transportMiddleware.PreapareTransportRequest(requestParameter: requestParameter, operationType: PNOperationType.PNGetChannelMembersOperation);
+			var transportResponse = await PubnubInstance.transportMiddleware.Send(transportRequest: transportRequest);
+			if (transportResponse.Error == null) {
+				var responseString = Encoding.UTF8.GetString(transportResponse.Content);
+				PNStatus errorStatus = GetStatusIfError(requestState, responseString);
+				if (errorStatus == null) {
+					PNStatus status = new StatusBuilder(config, jsonLibrary).CreateStatusResponse(requestState.ResponseType, PNStatusCategory.PNAcknowledgmentCategory, requestState, (int)HttpStatusCode.OK, null);
+					JsonAndStatusTuple = new Tuple<string, PNStatus>(responseString, status);
+				} else {
+					JsonAndStatusTuple = new Tuple<string, PNStatus>(string.Empty, errorStatus);
+				}
+				returnValue.Status = JsonAndStatusTuple.Item2;
+				string json = JsonAndStatusTuple.Item1;
+				if (!string.IsNullOrEmpty(json)) {
+					List<object> resultList = ProcessJsonResponse(requestState, json);
+					ResponseBuilder responseBuilder = new ResponseBuilder(config, jsonLibrary, pubnubLog);
+					PNChannelMembersResult responseResult = responseBuilder.JsonToObject<PNChannelMembersResult>(resultList, true);
+					if (responseResult != null) {
+						returnValue.Result = responseResult;
+					}
+				}
+			} else {
+				int statusCode = PNStatusCodeHelper.GetHttpStatusCode(transportResponse.Error.Message);
+				PNStatusCategory category = PNStatusCategoryHelper.GetPNStatusCategory(statusCode, transportResponse.Error.Message);
+				PNStatus status = new StatusBuilder(config, jsonLibrary).CreateStatusResponse(PNOperationType.PNGetChannelMembersOperation, category, requestState, statusCode, new PNException(transportResponse.Error.Message, transportResponse.Error));
+				returnValue.Status = status;
+			}
+			return returnValue;
+		}
+
+		private static string MapEnumValueToEndpoint(string enumValue)
+		{
+			string ret = "";
+			if (enumValue.ToLowerInvariant() == "custom") {
+				ret = "custom";
+			} else if (enumValue.ToLowerInvariant() == "uuid") {
+				ret = "uuid";
+			} else if (enumValue.ToLowerInvariant() == "uuid_custom") {
+				ret = "uuid.custom";
```

Comment:
What's the significance of changing the _ to . here?

Reply:
It's required conversation between C# request object and Rest api query parameter key values, So if something is going to change, we need to update this conversation.